tests/Support/ExtractsDataTest.php

```diff
@@ -26,12 +26,12 @@ public function testConvertsTimestampToExpectedFormat()
     public function testConvertsTimestampToUTC()
     {
         $dateTime = new DateTime('1 hour ago');
-        $dateTime->setTimezone(new DateTimeZone('Europe/Zurich'));
+        $dateTime->setTimezone(new DateTimeZone('Europe/Simferopol'));
 
         $this->assertGreaterThan(0, $dateTime->getOffset());
 
         $expectedUTC = clone $dateTime;
-        $expectedUTC->sub(date_interval_create_from_date_string('2 hours'));
+        $expectedUTC->sub(date_interval_create_from_date_string('3 hours'));
```

Comment:
Had to make a change here to a timezone that doesn't observe DST so we don't have moody tests